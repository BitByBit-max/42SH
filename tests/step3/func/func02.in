foo()
{
    bar()
    {
        echo "foobar"
    }
}

echo "calling foo"
foo

echo "calling bar"
bar
