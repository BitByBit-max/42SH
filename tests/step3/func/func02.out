calling foo
calling bar
foobar
